src/runMocha.js

```diff
@@ -57,9 +63,18 @@ const runMocha = ({ config, testPath, globalConfig }, workerCallback) => {
     rootDir: config.rootDir,
     collectCoverage: globalConfig.collectCoverage,
     coveragePathIgnorePatterns: config.coveragePathIgnorePatterns,
-    allowBabelRc: coverageOptions.useBabelRc
+    allowBabelRc: coverageOptions.useBabelRc,
   });
 
+  if (config.setupFilesAfterEnv) {
+    config.setupFilesAfterEnv.forEach(path => {
+      // eslint-disable-next-line global-require,import/no-dynamic-require
+      const module = require(path);
+      if (config.clearMocks && module.clearMocks) {
+        clearMocks = module.clearMocks;
```

Comment:
doesn't this mean that the last setup file to set clearMocks will wipe out any set by the previous files?

Reply:
Defining clearMocks export it several files hardly makes sense... But we can add a check and throw error if clearMocks is redefined.

Reply:
Alternatively, could we build up an array of functions, and call them all in order?

I think it makes sense that anything you can do in one file, you can refactor to do in multiple.